.gitignore

```diff
@@ -11,3 +11,4 @@ example/css/*.css
 dist/ui-coverage/
 npm-debug.log
 .publish
+.vscode
```

Comment:
Can we add a trailing whitespace here? :+1:

Reply:
added